runtime/dali/src/lib.rs

```diff
@@ -111,7 +111,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	//   `spec_version`, and `authoring_version` are the same between Wasm and native.
 	// This value is set to 100 to notify Polkadot-JS App (https://polkadot.js.org/apps) to use
 	//   the compatible custom types.
-	spec_version: 2000,
+	spec_version: 3000,
```

Comment:
should just bump impl version instead

Reply:
As we are including a new pallet, doesn't that mean we are changing the spec?

Reply:
well true